src/components/network-modification-tree-pane.js

```diff
@@ -152,9 +152,27 @@ export const NetworkModificationTreePane = ({
                         removeNotificationByNode(currentNodeRef.current?.id)
                     );
                 }
+                if (
+                    studyUpdatedForce.eventData.headers['nodes'].some(
+                        (nodeId) => nodeId === selectedNodeIdForCopy
+                    )
+                ) {
+                    setSelectedNodeIdForCopy(null);
+                    snackInfo(
+                        '',
+                        'Invalidation de la copie du nœud suite à modification'
+                    );
+                }
             }
         }
-    }, [studyUuid, studyUpdatedForce, updateNodes, dispatch]);
+    }, [
+        studyUuid,
+        studyUpdatedForce,
+        updateNodes,
+        dispatch,
+        selectedNodeIdForCopy,
```

Comment:
You will still fire this effect when the copied node id changes.
This is not what we expect.
This effect should only fire when there is a new notification to consume.